Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -178,6 +187,13 @@ namespace gdjs {
       return gdjs.multiplayerMessageManager.getNumberOfConnectedPlayers();
     };
 
+    /**
+     * Returns true if the player at this position is connected to the lobby.
+     */
+    export const isPlayerConnectedToLobby = (playerNumber: number) => {
```

Comment:
Is the notion of "lobby" really relevant here?

```suggestion
     * Returns true if the player at this position is connected.
     */
    export const isPlayerConnected = (playerNumber: number) => {
```